### ISSUE TITLE
Crash looping with UnhandledPromiseRejection

### ISSUE THREAD

**Issue by azafred** [hassio-addons/addon-node-red#1201]
# Problem/Motivation

The addon just auto-upgraded and is now crash looping with the following message:
```
4 Dec 17:41:51 - [debug] [server:Home Assistant] Services Loaded
4 Dec 17:41:51 - [red] Uncaught Exception:
4 Dec 17:41:51 - [error] UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "#<Object>".
[17:41:52] INFO: Starting Node-RED...
```
Note: I have log_level set to debug, not that it helps much...

## Expected behavior

For node-Red to not crash-loop and for the logs to show what the problem is instead of a generic error when there is an issue.

## Actual behavior

It crash-loops with no meaningful logging

## Steps to reproduce

If I know what the problem was, I would tell you! I am guessing, since it doesn't appear to be widespread, that it is one of the installed palette causing the problem, but which one is the question....

**Comment by aaamoeder** [comment]
I have the same error but it doesn't cause a reboot. (running version 10.1.1)
What version of the addon are you running ?

**Comment by azafred** [comment]
I am on 10.2.0. Previous version was working great, but I had the
auto-upgrade turned on, so from one day to the next all of my nodered
automation stopped working. Due to lack of debug information and the short
lived container, I have not been able to figure out what the problem is,
and at that point all I can think of doing is removing my flows file and
manually recreating the flows until I find if that is indeed the problem.
Not exactly an ideal situation…

**Comment by aaamoeder** [comment]
try restoring a backup of the previous version of NR (10.1.1) 
That "fixed" it for me, or at least the errors are not causing a boot loop anymore.
I've contacted @frenck about this on discord but he currently has no time to see to this issue.
Restoring the previous version should get you sorted ! ;-)

**Comment by azafred** [comment]
Yep, ended up restoring the previous version of the add-in. The error is still there, and there is definitely some insight into the current problem:
```
(node:512) UnhandledPromiseRejectionWarning: #<Object>
(Use `node --trace-warnings ...` to show where the warning was created)
(node:512) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 57)
(node:512) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.``` so yes... they were not lying ;p 
Now to find where the issue is coming from...

**Comment by pjgrun** [comment]
Having same problem. After upgrading to 10.2.0 nodered is not able to complete startup, and goes into endless restart loop.
In the logs I can see: 

![image](https://user-images.githubusercontent.com/89712102/145387327-c38ebf00-0094-46d0-8115-59e79389cec7.png)

After downgrading to 10.1.1, I have been able to recover operations, while I still get the following warning:

![image](https://user-images.githubusercontent.com/89712102/145388500-b7f625ee-0890-4ac2-ba6e-231b5f4c543a.png)
